### ISSUE TITLE
Стр. Продажа и Аренда (п.7): смена раскладки

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#23]
@vladimir075 @nadezdafrolova 

Макет: https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3034%3A1334
Задачи:

1. Настольное и мобильное представление: переверстать по макету (внимание: в мобильной версии добавилось переключение карта/список, одновременно они не видны).

Проблемы: 

1. Фильтры для настольной и мобильной версий различаются, придется дублировать (2 разных блока, один виден на настольной ширине, другой на мобильной).

Вропосы и уточнения:

1. Возможны проблемы реализации (механизм писал не я, придется его заимствовать, разобраться)
>Важно: изменить поведение карты. Сделать поведение как на карте [aibnb](https://www.airbnb.com/s/Lisbon--Portugal/homes?place_id=ChIJO_PkYRozGQ0R0DaQ5L3rAAQ&refinement_paths%5B%5D=%2Fhomes&checkin=2022-05-04&checkout=2022-05-25&adults=0&children=0&infants=0&pets=0&search_type=AUTOSUGGEST)-ховер на цену → увиличение лейбла с ценой →клик на цену   →открытие окна preview.
3. Вижу только новый элемент для объекта в списке (сердечко), укажите новые для карты.
>На этой странице [новые](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=2993%3A1273) элементы для списка и карты. Лейбл с ценой на карте нужно увеличить и изменить цвет, для удобства клика.

**Comment by vladimir075** [comment]
Да, то что переключение карта/список в мобильной это важно. Думаю к этому вопросов нет.
Относительно фильтров, как я понимаю, в мобильной объединено то, что в настольной "на виду" плюс то, что внутри выпадашки "more filters" Если прогнозируемый технические проблемы есть, наверное лучше по ним сделать совместный колл с Олегом и Николаем.
Относительно новых элементов ждем ответ от @nadezdafrolova

**Comment by nadezdafrolova** [comment]
Новыми элементами для всей страницы здесь https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=2993%3A1273

По сути это новый вид для карточек в писке, и новый вид карточки объекта на самой карте по клику.

**Comment by nicothin** [comment]
https://nicothin.pro/nesia/search.html

В настольной ширине подглючивает работа фильтров (я ее взял как есть, сам механизм и фильтры не менялись, при копировании оно перестает корректно работать, при реальном внедрении этой проблемы, видимо, не будет)

**Comment by vladimir075** [comment]
ок
